adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -1,5 +1,16 @@
 = sycl_khr_command_graph
 
+The graph abstraction defines an explicitly lazy execution model by decoupling
```

Comment:
If we put this in an `== Overview` sub-section header above this paragraph, it could help align more with the vendor extension if we ended up wanting to add sub-sub sections under it. In particular https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc#72-graph-building-mechanisms might be useful.